src/locales/ca/en/index.json

```diff
@@ -1,6 +1,8 @@
 {
-  "alreadyHaveAnAccount": "Already have an account? <a href='/'>Sign in here!</a>",
-  "dontHaveAnAccount": "Don’t have an account? <a href='/register'>Sign up here!</a>",
+  "alreadyHaveAnAccount": "Already have an account? <a href='?journey'>Sign in here!</a>",
```

Comment:
Similar to my below comment - the `Already have an account? Sign in here!` link doesn't seem to do anything, with just the URL updated.

Reply:
This should be fixed now. Thanks for catching it.